gusb/gusb-device.c

```diff
@@ -450,6 +466,27 @@ _g_usb_device_save(GUsbDevice *self, JsonBuilder *json_builder, GError **error)
 	return TRUE;
 }
 
+/**
+ * g_usb_device_get_created:
+ * @self: a #GUsbDevice
+ *
+ * Gets the date and time that the #GUsbDevice was created.
+ *
+ * This can be used as an indicator if the device replugged, as the vendor and product IDs may not
+ * change for some devices. Use `g_date_time_equal()` to verify equality.
+ *
+ * Returns: (transfer none): a #GDateTime
+ *
+ * Since: 0.4.5
+ **/
+GDateTime *
+g_usb_device_get_created(GUsbDevice *self)
```

Comment:
get_created_date_time?

Reply:
On the logic that we might have `g_usb_device_get_created_iso8601()` in the future?

Reply:
actually dunno, maybe just get_created is enough since from the datetime you can get the iso format